metadata-service/openapi-servlet/src/main/java/io/datahubproject/openapi/metrics/MicrometerController.java

```diff
@@ -0,0 +1,26 @@
+package io.datahubproject.openapi.metrics;
+
+import com.linkedin.metadata.utils.metrics.MetricUtils;
+import io.prometheus.client.exporter.common.TextFormat;
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@AllArgsConstructor
+@RequestMapping("/metrics")
+@Slf4j
+public class MicrometerController {
+
+    @GetMapping(value = "/prometheus")
```

Comment:
so if I understand it well, this will expose the same metrics on this endpoint too and the already existing JMX one (if enabled)?